app/helpers/employee_helper.rb

```diff
@@ -0,0 +1,20 @@
+module EmployeeHelper
+
+  def check_value(check)
+    if check
+      1
+    else
+      0
+    end
```

Comment:
In this condition can be use a ternary operator same like in at that your check_save conditional.

Reply:
I agree with that.